nominees/tamanu.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Tamanu",
+  "aliases": [
+    ""
+  ],
+  "description": "Tamanu is a free and open-source electronic health record (EHR) for low resource and remote settings.",
+  "website": "https://www.beyondessential.com.au/products/tamanu/",
+  "license": [
+    {
+      "spdx": "GPL-2.0",
+      "licenseURL": "https://github.com/beyondessential/tamanu-open/blob/master/license"
```

Comment:
LicenseURL field indicates a non-OSI license - [TAMANU OPEN SOURCE LICENSE](https://github.com/beyondessential/tamanu-open/blob/master/license). Digital public goods must demonstrate the use of an approved open license. For open-source software, only [OSI approved licenses](https://github.com/DPGAlliance/publicgoods-candidates/blob/main/help-center/licenses.md) are accepted.

Reply:
The DPGA relies on the [licenses](https://opensource.org/licenses/alphabetical) stewarded by the Open Source Initiative, the [current license](https://github.com/beyondessential/tamanu-open/blob/master/license) you are using poses some challenges for this compliance, it would be difficult to make an exception without risking impartiality of the DPG review team. OSI approved licenses are vetted licenses with robust process and expert knowledge behind them, as well as being well tested and established for different scenarios.

That said we would like to suggest a couple options that may be able to work for this digital solution to pass this indicator:

1. **The quick and easy way**: We would suggest defaulting to GPL-2.0 or [AGPL-3.0](http://agpl-3.0/) as the project license and may be consider setting up a manifesto (see example [here](https://hwb.ngo/manifesto/)) where you put out this intention for its use on health and education projects and in low and middle-income countries, of course this would not be a legal restriction that can be imposed (which is the main drawback).

2. **The not so easy way:** Submit their own license for OSI revision, this would require from your side to apply and go through the [license review process](https://opensource.org/approval). Given the [Open Source Definition](https://opensource.org/docs/osd) we think it would be hard to get it approved as it is right now, you would need to carefully state your conditions not as distribution restrictions, but if approved it could set a precedent and option for many other open source software in the future.